### ISSUE TITLE
Crash on tauri app after open midi file has "Pitch Bench" message

### ISSUE THREAD

**Issue by NDK2020** [webern/midi_file#17]
Hi,

I'm using this lib to read midi file data, and get error when open this midi file has "Pitch Bench" message

**To Reproduce**
open this midi file [LINK](https://drive.google.com/file/d/1KvdeLBql0tBg5EwRT7Ndxu9ttRrvrdoc/view?usp=sharing)

```
Thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Unimplemented { site: "/Users/khoa.nd/.cargo/registry/src/github.com-1ecc6299db9ec823/midi_file-0.0.3/src/core/message.rs:317", feature: "pitch bend: https://github.com/webern/midi_file/issues/10" })', src/main.rs:85:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
```

Can you suggests a way to ignore these unwanted message, currently I only need Note On/Off messages.

thx,
P/S: I'm new to rust

**Comment by NDK2020** [comment]
Update,

I temporarily by-pass the error by replace `impl!` by  `Ok(Message::Undefined1)` in  `message.rs`

 ```
           StatusType::PitchBend => {
                Ok(Message::Undefined1)
                // noimpl!("pitch bend: https://github.com/webern/midi_file/issues/10")
            }
```

**Comment by webern** [comment]
Were you using <=`v0.0.3`? If so, can you try `v0.0.4`? This was just implemented and released https://github.com/webern/midi_file/pull/15

**Comment by webern** [comment]
@NDK2020 is the linked midi file something that I can add to this git repository as a test? Do you own the copyright? If so can you grant permission to use it. I've been putting `.info` files next to the `.mid` files for attribution. For example: https://github.com/webern/midi_file/blob/f5374348727e42d37ad952c8c95cfe439309e693/tests/data/als_die_roemer.info#L1-L3

**Comment by webern** [comment]
Oh, I see #15 only added support for *writing*, I'll see if I can quickly add *reading* of pitch bend events.

**Comment by NDK2020** [comment]
> @NDK2020 is the linked midi file something that I can add to this git repository as a test? Do you own the copyright? If so can you grant permission to use it. I've been putting `.info` files next to the `.mid` files for attribution. For example:
> 
> https://github.com/webern/midi_file/blob/f5374348727e42d37ad952c8c95cfe439309e693/tests/data/als_die_roemer.info#L1-L3

sr for late reply, you can use it freely.

> Oh, I see https://github.com/webern/midi_file/pull/15 only added support for writing, I'll see if I can quickly add reading of pitch bend events.

great to have!!!! As I use midi files not for interact with instruments but for making rhythm game, therefore I'm implementing a small tool which use this repo to parse midi files for suitable txt format. Therefore if some midi events is not supports yet, I will send you as a test file.

**Comment by webern** [comment]
It turned out not to be so quick because I have trouble understanding the spec on this and I'm not 100% sure that this is correct: #15

What I plan to do is create a MIDI file with Logic Pro where I know the exact Pitch Bend values before reading the file, but I haven't gotten back to this.

**Comment by webern** [comment]
I did a little work on this yesterday. I have a hard time understand the spec and lsb/msb for these messages. But I might have it right on this branch? https://github.com/webern/midi_file/tree/bendy

**Comment by webern** [comment]
The part that is confusing me is that Logic Pro gives you 0-127 in the UI and the way I have it in my branch that actually corresponds to 0 - 16383 in chunks. In other words I think Logic Pro is writing that 0-127 in the MSB. The other application I used may have written numbers slightly different from what I saw in the UI. So I'm left a bit unsure about my implementation.

#10

**Comment by webern** [comment]
This is finally fixed in v0.0.5
